tests/PivotEventTraitTest.php

```diff
@@ -33,22 +37,35 @@ public function setUp()
     private function startListening()
```

Comment:
Maybe we can rename function to startListeningAndGetUser()  or something like that?

Reply:
I would suggest against that too keep the code more SOLID, (S - Single-responsiblity principle).
Also "startListening" is very verbose about what it is doing and `$user = User::first()` is very obvious knowing what it doing.
When I went to do the pull request I first was going about it slightly differently and the fact that `return User::first()` was in the function tripped me up slightly.

Reply:
SOLID vs DRY

Reply:
There are cases where `$this->startListening()` is called multiple times in a a test but the `return User::first()` is only used the first time it is called.
If you want to be more DRY, wouldn't it be better to have `$firstUser = User::find(1)` within the `setUp` function?